go.mod

```
@@ -10,7 +10,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/dynamodbstreams v1.25.1
 	github.com/aws/smithy-go v1.22.3
 	github.com/conduitio/conduit-commons v0.5.1
-	github.com/conduitio/conduit-connector-sdk v0.12.0
+	github.com/conduitio/conduit-connector-sdk v0.13.0
```

the most recent is 0.13.2
```suggestion
	github.com/conduitio/conduit-connector-sdk v0.13.2
```

---

Updated.